app/scripts/utils/functional.es6

```diff
@@ -36,6 +60,36 @@ function asArray(obj) {
     .map(k => [k, obj[k]]);
 }
 
+const hasProperty = (data, propName) => (!isUndefined(data) && isObject(data) && !isUndefined(data[propName]));
```

Comment:
Ooh, there's a lodash function for this: `lodash/object/has` -- `has(data, propName)`